src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -74,7 +78,7 @@ object Hdfs {
 }
 
 // IndexLogEntry-specific Source that uses SparkPlan as a plan and Hdfs as data.
-case class Source(plan: SparkPlan, data: Seq[Hdfs])
+case class Source(plan: SparkPlan, data: Seq[Hdfs], relations: Seq[RelationMetadataEntry])
```

Comment:
how about putting this under `SparkPlan.Properties` (same place as `rawPlan`)?

Reply:
This is not relevant after https://github.com/microsoft/hyperspace/pull/99/files#r463881967